fission-cli/library/Fission/CLI/Types.hs

```diff
@@ -1,4 +1,3 @@
-{-# LANGUAGE AllowAmbiguousTypes  #-}
```

Comment:
Unused. This extension is less scary than it sounds, but also not actually used, so 🔪